infrastructure/helm-chart/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Airy Core
-name: helm-charts
-version: 0.5.0
+name: airy
+version: 0-develop
```

Comment:
The version of the helm-charts is stamped at CI time. However, a version for the chart must be specified and it must start with a digit. So, I'm not sure what is the best value here.